### ISSUE TITLE
Documentation broken links in Release notes for Corda 4

### ISSUE THREAD

**Issue by JoseAntonioBarriga** [corda/corda#5204]
In the Release notes for Corda 4 document are many "...to learn more..." links and none of the work. Is there a new version of this document? Please send me the link information.

**Comment by gendal** [comment]
what document are you looking at?  I am looking here: https://docs.corda.net/release-notes.html. and the links I tried work.  Is there another URL we've not updated properly?

**Comment by JoseAntonioBarriga** [comment]
Yes, it is, unfortunately, the one that is published in Github (https://github.com/corda/corda/blob/release/4/docs/source/release-notes.rst#id29) has only broken links. It seems that many of the links written in the GitHub repository are broken.

Thanks for your prompt help!


José Antonio Barriga
CEO
Movil: +569 7757 8444
Email: joseantonio.barriga@gateway-dlt.cl<mailto:joseantonio.barriga@gateway-dlt.cl>

[PieDePagina]

De: Richard Gendal Brown [mailto:notifications@github.com]
Enviado el: viernes, 7 de junio de 2019 12:31
Para: corda/corda <corda@noreply.github.com>
CC: JoseAntonioBarriga <joseantonio.barriga@live.com>; Author <author@noreply.github.com>
Asunto: Re: [corda/corda] Documentation broken links in Release notes for Corda 4 (#5204)


what document are you looking at? I am looking here: https://docs.corda.net/release-notes.html. and the links I tried work. Is there another URL we've not updated properly?

**Comment by gendal** [comment]
ah... ok - I see the problem.  That's an .rst file.  GitHub tries to render it but doesn't do an amazing job.  We process those files to build the docsite: https://docs.corda.net/building-the-docs.html

In other words, it's not expected that the links will work when viewed via the GitHib renderer.

**Comment by JoseAntonioBarriga** [comment]
Thanks any way…I will keep in touch since my children´s breakfast, lunch and dinner is relying on Corda!

José Antonio Barriga
CEO
Movil: +569 7757 8444
Email: joseantonio.barriga@gateway-dlt.cl<mailto:joseantonio.barriga@gateway-dlt.cl>

[PieDePagina]

De: Richard Gendal Brown [mailto:notifications@github.com]
Enviado el: viernes, 7 de junio de 2019 14:25
Para: corda/corda <corda@noreply.github.com>
CC: JoseAntonioBarriga <joseantonio.barriga@live.com>; Author <author@noreply.github.com>
Asunto: Re: [corda/corda] Documentation broken links in Release notes for Corda 4 (#5204)


ah... ok - I see the problem. That's an .rst file. GitHub tries to render it but doesn't do an amazing job. We process those files to build the docsite: https://docs.corda.net/building-the-docs.html

In other words, it's not expected that the links will work when viewed via the GitHib renderer.

**Comment by JoseAntonioBarriga** [comment]
Richard, this link https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases/net/corda/corda-network-builder/4.0-corda/corda-network-builder-4.0-corda-executable.jar which is taken from the cords docs site, is broken. Can you redirect me to the real url?
Thanks!

José Antonio Barriga

________________________________

**Comment by gendal** [comment]
Hi José - you're referring to issue #5203, right? (https://github.com/corda/corda/issues/5203).  I'll comment there too but tl;dr is sorry... we're working on it!
